giranezafiacre/My-Brand-Giraneza-Fiacre
# chore to finish firebase contact form integration

#### What does this PR do?
-it integrates firebase in frontend in the contact form

#### Description of Task to be completed?
-to enable storing of contact information of a user

#### How should this be manually tested?
-to download codes and run index.html and go in the contact form to store data and check-in the databaseUrl:"https://contactform-11c23.firebaseio.com"

#### Any background context you want to provide?
-N.A
#### What are the relevant pivotal tracker stories?
-N/A
#### Screenshots (if appropriate)
-N/A
#### Questions:
-N/A
